docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+          won't add that component without first removing the entry from the
+          values file.</para>
+        </listitem>
+
+        <listitem>
+          <para>You can supply multiple values files and they can be
+          combined.</para>
+        </listitem>
+
+        <listitem>
+          <para>When there are 2 yaml files, if there is a matching entry
+          (anything other than a dictionary) the value from 2nd file will
+          overwrite the first.</para>
+        </listitem>
+
+        <listitem>
+          <para>Dictionaries will always be merged.</para>
+        </listitem>
+      </itemizedlist>
+
+      <para>For instance, you want to update logging detail. You could have
+      another file to update that value, or any other list value using an
+      override yaml file. That override yaml file would be considered a second
+      file in addition to the values.yaml file.</para>
+
+      <para>As we will see later, components are defined as lists, so any
+      definition of a component in a user values file will replace all
+      instances of the component in the default chart. Other options (for
+      instance configuring the costs for cpu or file access) are implemented
+      as a dictionary, so options can be selectively set in a users values
+      file, and the other options will be retained.</para>
+
+      <sect3 id="merging_AND_Overrides">
+        <title>Merging and Overriding</title>
+
+        <para>Having multiple yaml files, such as one for logging, another for
+        storage, yet another for secrets and so forth, the files can be in
+        version control, checked in, etc. and has the benefit of only
+        defining/changing the specific area required, while ensuring any
+        non-changing areas are left untouched. The rule here to keep in mind
+        where multiple yaml files are applied, the later ones will always
+        overwrite the values in the earlier ones. They are merged in in
+        sequence.</para>
+
+        <para>Another point to consider, where there is a global dictionary
+        such as root: and its value redefined in the 2nd file (as a
+        dictionary) it would not be overwritten. There is no “nice way” to
+        replace a dictionary, you cant overwrite a dictionary. You can
+        redefine a dictionary and set it to null, which will effectively wipe
+        it out.</para>
+
+        <para><emphasis role="bold">WARNING</emphasis>: If you had a global
+        definition (such as Storage: Planes:) and merge it where that becomes
+        redefined it would wipe out every definition in the list.</para>
+
+        <para>Another means to wipe out every value in a list is to pass in an
+        empty set denoted by a [ ] such as this example:</para>
+
+        <para><programlisting>bundles: []</programlisting>This would wipe out
+        any properties defined for bundles.</para>
+      </sect3>
+
+      <sect3 id="GenerallyApplicable">
```

Comment:
Not sure why this is here - what do name/prefix/service refer to?  Should include examples.  And yaml files merged in order is a Helm feature covered above

Reply:
should I just remove this section <sect3> entirely?